README.md

```diff
@@ -279,16 +279,26 @@ superpowers:
 Configuration
 -------------
 
+### Configuration file
+
+Hub stores its configuration in a file named "hub", located in a directory
+determined in accordance with the 
+[XDG Base Directory Specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html).
```

Comment:
Sorry, I just moved house and have had intermittent internet access. I thought I'd get a chance to do this before I moved, but it didn't happen. Lucky this is not very important :smile: I'll sort it out when I get a chance over the next couple of days.